### PR TITLE
Docs: Gen maintenance fixes for September

### PR DESCRIPTION
**Pull request by owans**
## What
- Add hackathon cheatsheet
- Update MetaMask page to remove ref to RPC API 

## Why
- Related to #102 
- Hackathon resources

## Refs
- [Task](https://rsklabs.atlassian.net/browse/DV-204?atlOrigin=eyJpIjoiZWZiN2MzZDZmMDljNGFmNDlkOTk3YzRhMzc2ZGIxMGQiLCJwIjoiaiJ9)

### CODE REVIEW

File: docs/02-developers/02-requirements/index.md [hunk 1]
```diff
@@ -8,8 +8,14 @@ description: "Minimum hardware requirements for Rootstock."
...
 
+:::tip[Hackathon Info]
+
+Participating in a Rootstock hackathon or Workshop? View the [Hackathon Resources](/resources/hackathon/) section.
+
+:::
+
 ## Solidity Version
-- Supported solc version: `0.8.19`
+- Supported solc version: `0.8.29`
```
Comment (ivegabr) [inline docs/02-developers/02-requirements/index.md]:
```suggestion
- Supported solc version: `0.8.20`
```

File: docs/04-resources/07-hackathon/index.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ tags: [hackathons, rsk, workshop, resources, rootstock]
 description: "Hackathon resources and tools"
 ---
 
-This guide details the necessary hardware and software requirements for developing on the Rootstock blockchain. It includes setup instructions for essential tools such as Java, Node.js, Hardhat, and RSKj, ensuring developers have a clear path to prepare their environment for Rootstock projects, whether for local development, testing, or deployment.
+This guide details the necessary hardware and software requirements for developing on the Rootstock blockchain. It includes setup instructions for essential tools such as Java, Node.js, Hardhat, and RSKj, ensuring developers have a clear path to prepare their environment for Rootstock projects, whether for local development, testing, or deployment. Download the [Hackathon Cheatsheet PDF Here](https://dev.rootstock.io/static/rootstock-cheatsheet.pdf)
```
Comment (ivegabr) [inline docs/04-resources/07-hackathon/index.md]:
```suggestion
This guide details the necessary hardware and software requirements for developing on the Rootstock blockchain. It includes setup instructions for essential tools such as Java, Node.js, Hardhat, and RSKj, ensuring developers have a clear path to prepare their environment for Rootstock projects, whether for local development, testing, or deployment. Download the [Hackathon Cheatsheet PDF Here](/rootstock-cheatsheet.pdf)
```

**Review by ivegabr — CHANGES_REQUESTED** [review]
made 2 changes

**Comment by owans** [comment]
Changes implemented. Thanks @ivegabr

**Review by ivegabr — APPROVED** [review]
ok

**Pull request merged**